sections/main-product.liquid

```diff
@@ -106,6 +106,51 @@
                   {{ form | payment_terms }}
                 {%- endform -%}
               </div>
+            {%- when 'inventory' -%}
+              <p
+                class="product__inventory no-js-hidden{% if block.settings.text_style == 'uppercase' %} caption-with-letter-spacing{% elsif block.settings.text_style == 'subtitle' %} subtitle{% endif %}{% if product.selected_or_first_available_variant.inventory_management != 'shopify'%} visibility-hidden{% endif %}"
+                {{ block.shopify_attributes }}
+                id="Inventory-{{ section.id }}"
+                role="status"
+              >
+                {%- if product.selected_or_first_available_variant.inventory_quantity != 0 and product.selected_or_first_available_variant.inventory_management == 'shopify' -%}
```

Comment:
As we discussed, let's do this improvement latter after we hear opinion on this from different devs. Right now I'd prefer to move forward, since it's not a technical issue.